packages/web/src/pages/oauth-login-page/OAuthLoginPage.tsx

```diff
@@ -64,29 +73,47 @@ export const OAuthLoginPage = () => {
 
   const clearErrors = () => {
     setGeneralSubmitError(null)
-    setHasCredentialsError(false)
+    setSignInError(null)
   }
 
-  const setAndLogGeneralSubmitError = useCallback(
-    (isUserError: boolean, errorMessage: string, error?: Error) => {
-      setGeneralSubmitError(errorMessage)
-      record(
-        make(Name.AUDIUS_OAUTH_ERROR, {
-          isUserError,
-          error: errorMessage,
-          appId: (apiKey || appName)!,
-          scope: scope!
-        })
-      )
-      if (error && !isUserError) {
-        reportToSentry({ level: ErrorLevel.Error, error })
-      }
-    },
-    [record, appName, apiKey, scope]
-  )
+  const toggleOtpUI = (on: boolean) => {
+    if (on) {
+      setShowOtpInput(true)
+      setSignInError(messages.otpError)
+    } else {
+      setSignInError(null)
+      setShowOtpInput(false)
+    }
+  }
+  useEffect(() => {
```

Comment:
why is this effect necessary versus just setting to show the otp input after we see a 403? is it to clear state if the user changes the email? if so, maybe that input should call a function to clear the error instead

Reply:
yeah to clear it if they change the email. ill have the input do this instead 👍